### DISCUSSION TITLE
Unsubscribing for users that "do" the mutation on the client

### DISCUSSION THREAD

**Discussion by Miloshinjo** [urql-graphql/urql#1951] [Q&A] [answered]
Is there a way of not triggering subscriptions or ignoring them for the user that is currently performing the mutation.

Common case is that user will mutate something, it will be handled in the `updates` or `optimistic`, and then subscription will come, but it's gonna be redundant for that user only. 

Is there a way to turn it off?

**Comment by JoviDeCroock** [comment, marked as answer]
You would have to filter that on your server probably, or you can check it in code if the payload has an indicator of that, essentially though it will be a duplicate update so it won't have an effect as the subscription payload is already applied by means of the mutation response

**Reply by Miloshinjo** [reply]
Thank you!
